gpii/node_modules/gpii-oauth2/gpii-oauth2-authz-server/src/AuthServer.js

```diff
@@ -4,29 +4,21 @@ Copyright 2014-2017 OCAD university
 Licensed under the New BSD license. You may not use this file except in
 compliance with this License.
 
-The research leading to these results has received funding from the European Union's
-Seventh Framework Programme (FP7/2007-2013) under grant agreement no. 289016.
```

Comment:
@simonbates, the removal of the EU funding notice from auth server files should be fine as they are exclusively worked by you and me.

Reply:
Antranig and Kasper are listed in the contributors for `AuthServer.js`:

    $ git shortlog --summary gpii/node_modules/gpii-oauth2/gpii-oauth2-authz-server/src/AuthServer.js
        10  Antranig Basman
        55  Cindy Qi Li
         1  Justin Obara
         1  Kasper Markus
        32  Simon Bates